django_dbq/models.py

```diff
@@ -50,24 +45,37 @@ def delete_old(self):
         """
         Delete all jobs older than DELETE_JOBS_AFTER_HOURS
         """
-        delete_jobs_in_states = [Job.STATES.FAILED, Job.STATES.COMPLETE]
+        delete_jobs_in_states = [Job.State.FAILED, Job.State.COMPLETE]
         delete_jobs_created_before = datetime.datetime.utcnow() - datetime.timedelta(hours=DELETE_JOBS_AFTER_HOURS)
         logger.info("Deleting all job in states %s created before %s", ", ".join(delete_jobs_in_states), delete_jobs_created_before.isoformat())
         Job.objects.filter(state__in=delete_jobs_in_states, created__lte=delete_jobs_created_before).delete()
 
     def to_process(self, queue_name):
-        return self.select_for_update().filter(queue_name=queue_name, state__in=(Job.STATES.READY, Job.STATES.NEW))
+        return self.select_for_update().filter(queue_name=queue_name, state__in=(Job.State.READY, Job.State.NEW))
 
 
 class Job(models.Model):
 
-    STATES = Choices("NEW", "READY", "PROCESSING", "FAILED", "COMPLETE")
+    class State:
+        NEW = 'NEW'
+        READY = 'READY'
+        PROCESSING = 'PROCESSING'
+        FAILED = 'FAILED'
+        COMPLETE = 'COMPLETE'
+
+    STATES = [
```

Comment:
Can we make this backwards compatible with how it used to work, ie set `STATES = State`, and the choices as `STATES_CHOICES` or something?